src/lib/docs/Templates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Templates Overview
+description: An overview of the available WharfKit templates, designed to accelerate your blockchain development with Antelope.
+category: Templates
+published: true
+slug: index
+---
+
+# WharfKit Templates Overview
+
+WharfKit provides a suite of templates which empower developers to easily create custom plugins that enhance and extend the capabilities of the WharfKit ecosystem.
+
+Here's an overview of the available templates:
+
+## [Account Creation Plugin Template](/docs/templates/account-creation-plugin-template)
+
+This template can be used as a starting point for developers looking to create [account creation plugins](/docs/session-kit/plugin-account-creation) that handle account creation. Through those plugins, developers can make any account creation service usable with the [Wharf SessionKit](/docs/session-kit).
```

Comment:
When linking to these things, what I've been trying to do is only link to the component name itself. So a link to the `SessionKit` would only link on the keyword `SessionKit`.

Reply:
Also starting each section with "This template can be used as a starting point for developers" is repetitive and probably not useful. If we're repeating it over and over, maybe it can go into the intro and be omitted from each section?